app/containers/Wallet.jsx

```diff
@@ -0,0 +1,102 @@
+import ReactTimeout from "react-timeout";
+import { Route } from "react-router-dom";
+import { AnimatedSwitch } from "react-router-transition";
+import { StaticSwitch } from "shared";
+import HomePage from "components/views/HomePage/HomePage";
+import SettingsPage from "components/views/SettingsPage/SettingsPage";
+import AccountsPage from "components/views/AccountsPage/AccountsPage";
+import WalletError from "components/views/WalletError";
+import ErrorScreen from "components/ErrorScreen";
+import InvalidRPCVersion from "components/views/InvalidRPCVersion";
+import PrivacyPage from "components/views/PrivacyPage/PrivacyPage";
+import TransactionsPage from "components/views/TransactionsPage/TransactionsPage";
+import TransactionPage from "components/views/TransactionPage/TransactionPage";
+import TicketsPage from "components/views/TicketsPage/TicketsPage";
+import TutorialsPage from "components/views/TutorialsPage/TutorialsPage";
+import GovernancePage from "components/views/GovernancePage/GovernancePage";
+import ProposalDetailsPage from "components/views/ProposalDetailsPage/ProposalDetailsPage";
+import TrezorPage from "components/views/TrezorPage";
+import LNPage from "components/views/LNPage";
+import SideBar from "components/SideBar/SideBar";
+import { BlurableContainer } from "layout";
+import { useWallet } from "./hooks";
+import { useTheming, useMountEffect } from "hooks";
+
+const pageAnimation = {
+  atEnter: { opacity: 0 },
+  atLeave: { opacity: 0 },
+  atActive: { opacity: 1 }
+};
+
+const Wallet = ({ setInterval }) => {
+  const {
+    getPeerInfo,
+    expandSideBar,
+    politeiaEnabled,
+    compareInventory
+  } = useWallet();
+
+  const { uiAnimations } = useTheming();
+
+  // Notice that we return a cleanup logic as a function in useEffect/useMountEffect
+  // which will run on unmount.
+  useMountEffect(() => {
+    // Compare politeias inventory and update proposal list if they are different
+    // every 1 minute.
+    const piInvInterval = setInterval(() => {
+      if (politeiaEnabled) {
+        compareInventory();
+      }
+    }, 60000);
+
+    // Get peer info every 10 seconds, so we can no if there are no available
+    // peers.
+    const peerInfoInterval = setInterval(() => {
+      if (politeiaEnabled) {
```

Comment:
Is `politeiaEnabled` have to be `true` here for getting peer info?

Reply:
No it shouldn't good catch!

Reply:
@amassarwi

Reply:
that's was the original condition, will get rid of it